apps/metadata-editor/src/app/dashboard/dashboard-menu/dashboard-menu.component.html

```diff
@@ -30,26 +30,6 @@
       <ng-icon name="iconoirCalendar"></ng-icon>
       <span class="text-sm" translate="">dashboard.catalog.calendar</span>
     </a> -->
-    <a
```

Comment:
Let's also clean the commented code above. The versioning tools are meant for this.